### PR TITLE
Simplify store creation

### PR DESCRIPTION
**Pull request by banciur**
Tried to enable `redux-devtools-extension` I failed (issue will be created) but at least I cleaned a bit `index.js`.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -31,5 +32,5 @@ const render = (store) => {
     </Provider>, root);
 };
 
-export const store = compose(applyMiddleware(thunk))(createStore)(reducer);
+const store = createStore(reducer, applyMiddleware(thunk));
```
Comment (krzkaczor) [inline src/index.js]: TBH: i would leave `compose` as it was before - it's easier to add new middlewares in future.

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
+/* global document */
```
Comment (krzkaczor) [inline src/index.js]: do we really need it? what value does it bring?

Comment (banciur) [inline src/index.js]: Without it eslint returns error that global is undefined variable - [no-undef](http://eslint.org/docs/rules/no-undef)

File: src/index.js (hunk 1 above)
Comment (banciur) [inline src/index.js]: I thought you can pass multiple middelawares to applyMiddleware function

File: src/index.js (hunk 2 above)
Comment (krzkaczor) [inline src/index.js]: Plase set environement as browser instead: https://stackoverflow.com/questions/41858052/solving-linter-error-no-undef-for-document

File: src/index.js (hunk 1 above)
Comment (krzkaczor) [inline src/index.js]: You can, nevermind ;)

**Pull request merged**